README.md

```diff
@@ -189,9 +189,50 @@ Donation totals:
 Thanks for visiting!
 
 Jeff Johnson  
-jeff@digitalruby.com  
-http://www.digitalruby.com
+<jeff@digitalruby.com>  
+<http://www.digitalruby.com>
 
 [nuget]: https://www.nuget.org/packages/DigitalRuby.ExchangeSharp/
 [websocket4net]: https://github.com/kerryjiang/WebSocket4Net
- 
+[ApolloX]: src/ExchangeSharp/API/Exchanges/ApolloX/
+[Aquanow]: src/ExchangeSharp/API/Exchanges/Aquanow/
+[Binance]: src/ExchangeSharp/API/Exchanges/Binance/
+[Binance Jersey]: src/ExchangeSharp/API/Exchanges/BinanceGroup/
+[Binance.US]: src/ExchangeSharp/API/Exchanges/BinanceGroup/
+[Binance DEX]: src/ExchangeSharp/API/Exchanges/BinanceGroup/
+[BitBank]: src/ExchangeSharp/API/Exchanges/BitBank/
+[Bitfinex]: src/ExchangeSharp/API/Exchanges/Bitfinex/
+[Bitflyer]: src/ExchangeSharp/API/Exchanges/Bitflyer/
+[Bithumb]: src/ExchangeSharp/API/Exchanges/Bithumb/
+[BitMEX]: src/ExchangeSharp/API/Exchanges/BitMEX/
+[Bitstamp]: src/ExchangeSharp/API/Exchanges/Bitstamp/
+[Bittrex]: src/ExchangeSharp/API/Exchanges/Bittrex/
+[BL3P]: src/ExchangeSharp/API/Exchanges/BL3P/
```

Comment:
Thanks for adding this section. Do you think it will make the `README.md` harder to maintain? (as in it adds another area we have to keep updated)
I have been looking at this codebase for many years, so I don't have a perspective of how obvious it is to find the implementations for each exchange. As a relative newcomer, did you feel that it was difficult getting started?

Reply:
I did not but i am a fairly seasoned dev and am, more or less, a polyglot when it comes to programming languages so i'm not a very good litmus test. I think we could drop the links if maintenance is harder, wouldn't want that chore

Reply:
Do you mind removing it? I think it was a great thought though.

Reply:
Perhaps if you could think of a way to point to the directory in general, then we could add that link for ppl to start from.

Reply:
Done and done